modules/age.nix

```diff
@@ -60,18 +67,20 @@ let
   '') cfg.identityPaths;
 
   cleanupAndLink = ''
-    _agenix_generation="$(basename "$(readlink ${cfg.secretsDir})" || echo 0)"
-    (( ++_agenix_generation ))
+    if (( _localstatus > 0 )); then
+      mv "${cfg.secretsMountPoint}/$_agenix_generation"{,-incomplete}
+      _agenix_generation+=-incomplete
+    fi
```

Comment:
What is this about? Maybe worth a comment in the code.

Reply:
Done.

Reply:
`_localstatus` is defined [here](https://github.com/NixOS/nixpkgs/blob/6527928d394c236cc0a5d98902ae8eb7e5830374/nixos/modules/system/activation/activation-script.nix#L11).